tests/integration/config/plugins/integration-tests/data/tests/tests_illegal_atrocity.txt

```diff
@@ -316,33 +316,16 @@ test "Launch Quarg Scanning Mission And Board"
 		# board the ship so we're sure we get close to it to get scanned
 		input
 			command board
-		# wait for 7.5 seconds so it should be repaired and have scanned us
-		apply
-			"test: steps to wait" = 7 * 60 + 30
 		label "floating"
 		# empty input to make time pass
 		input
-		apply
-			"test: steps to wait" = "test: steps to wait" - 1
+		# wait for it to become hostile from the scanning
 		branch "floating"
-			"test: steps to wait" > 0
+			"reputation: Quarg" >= 0
+			"unpaid fines" == 0
 		input
 			key "return"
 
-test "Boarding Test - Repair Friendly Ship"
```

Comment:
is this test unsalvageable?

Reply:
I just removed it because it isnt really useful - it was intended to be a substep of the other two but if they fail it's probably due to the same thing anyway